src/main/java/pl/confitura/jelatyna/admin/Controller.java

```diff
@@ -0,0 +1,54 @@
+package pl.confitura.jelatyna.admin;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.confitura.jelatyna.common.TokenGenerator;
+import pl.confitura.jelatyna.email.EmailService;
+
+import javax.validation.Valid;
+import java.util.List;
+
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+import static org.springframework.web.bind.annotation.RequestMethod.PUT;
+
+@RestController("adminController")
+@RequestMapping("/api/admin")
+public class Controller {
```

Comment:
Inne servisy nie są HATEOS bo ... innych serwisów nie mamy. mamy tylko czyste repo. Moim zdaniem nie ma najmniejszego sensu żebyśmy umieszczali linki w naszym restowym API. Wszak tylko gui, i to świadomie, będzie z tego korzystał.

co do `/api` to dodam.